storage/rocksdb/ha_rocksdb.cc

```diff
@@ -4065,7 +4065,7 @@ class Rdb_trx_info_aggregator : public Rdb_tx_list_walker {
         {rocksdb::Transaction::AWAITING_PREPARE, "AWAITING_PREPARE"},
         {rocksdb::Transaction::PREPARED, "PREPARED"},
         {rocksdb::Transaction::AWAITING_COMMIT, "AWAITING_COMMIT"},
-        {rocksdb::Transaction::COMMITED, "COMMITED"},
+        {rocksdb::Transaction::COMMITTED, "COMMITTED"},
```

Comment:
This is causing a compilation failure. I think that we cannot change the `enum` literal name, because it is defined in a submodule (`include/rocksdb/utilities/transaction.h`).

Reply:
Hi,
not sure about my rebase command...
Regarding rocksdb I can prepare a PR for them, they are pretty reactive but I am not sure how this submodule is imported in mariadb 10.3 branch?

Reply:
I think that the submodule would have to be updated in the 10.2 branch and then merged up to 10.3. Maybe it would be best to submit this part of the patch in a separate pull request against 10.2. @spetrunia should know how often we update the RocksDB submodule. We might not want to update to the ‘bleeding edge’ without running some extensive tests first.

Reply:
Ok, that seems good to me and I have to check with rocksdb when mi PR are going to be merged, They seems to have been accepted...
https://github.com/facebook/rocksdb/pull/4827
https://github.com/facebook/rocksdb/pull/4813

Let's see if CI success (I made a mistake yesterday in my first rollback commit).